src/main/java/nextstep/sessions/domain/SessionStatus.java

```diff
@@ -0,0 +1,9 @@
+package nextstep.sessions.domain;
+
+public enum SessionStatus {
+  READY, RECRUITING, END;
+
+  public static boolean isRecruiting(SessionStatus status) {
+    return RECRUITING.equals(status);
+  }
```

Comment:
```suggestion
  public boolean isRecruiting() {
    return this == RECRUITING;
  }
```

클래스 메서드가 아닌 인스턴스 메서드로 구현 가능
enum도 객체이니 객체에 메시지를 보내면 된다.